ci/e2e-tests.yml

```diff
@@ -0,0 +1,112 @@
+name: 'dhis2: e2e tests'
+
+# Requirements:
+#
+# - Secrets:
+#     GITHUB_TOKEN
+#     CYPRESS_DHIS2_USERNAME
+#     CYPRESS_DHIS2_PASSWORD
+#
+# - Customize environment variables:
+#     BASE_URL_INSTANCES: Set the base url for the instances, e.g. https://test.e2e.dhis2.org
+#     NAME_PATTERN_LEGACY_INSTANCES: Set the name pattern for your instances. {version} will be replaced by "[majorVersion].[minorVersion]"
+#     NAME_PATTERN_DEV_INSTANCE: Set the name pattern for your dev instance. {version} will be replaced by "dev"
+#
+# - Other optional customizations:
+#     containers: Set the matrix containers array for the cypress job. The number of parallel Cypress instances running for each backend version will equal the array length.
+#
+# - Set status check as required:
+#     We recommend setting "e2e-tests-success"  as a required step in your workflow.
+
+on:
+    pull_request:
+        types: [opened, synchronize, reopened, ready_for_review]
+
+env:
+    BASE_URL_INSTANCES: https://test.e2e.dhis2.org
+    NAME_PATTERN_LEGACY_INSTANCES: '{version}dev'
+    NAME_PATTERN_DEV_INSTANCE: dev
+
+concurrency:
+    group: ${{ github.workflow}}-${{ github.ref }}
+    cancel-in-progress: true
+
+defaults:
+    run:
+        shell: bash
+
+jobs:
+    compute-versions:
+        if: github.event.pull_request.draft == false
+        runs-on: ubuntu-latest
+        outputs:
+            versions: ${{ steps.legacy-versions.outputs.versions }}
+        steps:
+            - name: compute-instance-url
+              id: instance-url
+              run: |
+                  url=${BASE_URL_INSTANCES%/}/${NAME_PATTERN_DEV_INSTANCE/"{version}"/dev}
+                  echo "url=$url" >> $GITHUB_OUTPUT
+                  echo "url: $url"
+
+            - uses: actions/checkout@v3
+
+            - id: legacy-versions
+              uses: dhis2/action-supported-legacy-versions@v1
+              with:
+                  instance-url-latest: ${{ steps.instance-url.outputs.url }} # can be removed if maxDHIS2Version has been specified
+                  username: ${{ secrets.CYPRESS_DHIS2_USERNAME }} # can be removed if maxDHIS2Version has been specified
+                  password: ${{ secrets.CYPRESS_DHIS2_PASSWORD }} # can be removed if maxDHIS2Version has been specified
+
+    cypress:
+        needs: compute-versions
```

Comment:
Should this job also have the same `if` as on line 40? i.e., will this job show as `waiting` or `skipped` if the `compute-versions` is skipped?

Reply:
`needs` should be enough, there is an implicit `success()` condition. Will show as `skipped` if `compute-versions` is skipped.